src/wp-includes/user.php

```diff
@@ -1826,6 +1826,34 @@ function sanitize_user_field( $field, $value, $user_id, $context ) {
 	return $value;
 }
 
+/**
+ * Adds any users from the given IDs to the cache that do not already exist in cache.
+ *
+ * @since 6.1.0
+ * @access private
+ *
+ * @see update_user_caches()
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
+ * @param array $ids               ID list.
```

Comment:
```suggestion
 * @param int[] $user_ids          Array of user IDs.
```

And rename the variable in code to ;)